### PR TITLE
[tellstick] Made constants and handler packages internal

### PR DESCRIPTION
**Pull request by VelinYordanov**
See eclipse/smarthome#4330

Signed-off-by: VelinYordanov <velin.iordanov@gmail.com>

### CODE REVIEW

**Comment by wborn** [comment]
We might want to merge #2994 first.

**Comment by wborn** [comment]
Since #2994 got merged I've rebased this PR to resolve the merge conflicts @martinvw, @cweitkamp.

File: addons/binding/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/discovery/TellstickDiscoveryService.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
  * @author Jarle Hjortland - Initial contribution
  */
 public class TellstickDiscoveryService extends AbstractDiscoveryService
-        implements org.openhab.binding.tellstick.handler.DeviceStatusListener {
+        implements org.openhab.binding.tellstick.internal.handler.DeviceStatusListener {
```
[the remark this answers is not in the document]
Comment (wborn) [reply addons/binding/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/discovery/TellstickDiscoveryService.java]: There was no reason so I've imported the class intead!

[the remark this answers is not in the document]
Comment (cweitkamp) [reply addons/binding/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/discovery/TellstickDiscoveryService.java]: Great. Thanks.

**Pull request merged**